.golangci.yaml

```diff
@@ -7,68 +7,8 @@ run:
   timeout: 2m
 
 linters:
-  disable-all: true
```

Comment:
Can we leave these here commented out? the idea is to slowly add them as we see fit

Reply:
Right now, all linters are passing except for the `unused` linter. You wanted to keep the linters disabled and enable things individually instead?

Reply:
Ahh, let's leave it as you have it, and we can check how to get it fixed later.